src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -2035,11 +2035,15 @@ namespace GraphQL.Types
         public virtual object? ParseLiteral(GraphQLParser.AST.GraphQLValue value) { }
         public abstract object? ParseValue(object? value);
         public virtual object? Serialize(object? value) { }
+        [System.Diagnostics.CodeAnalysis.DoesNotReturn]
         protected GraphQLParser.AST.GraphQLValue ThrowASTConversionError(object? value) { }
+        [System.Diagnostics.CodeAnalysis.DoesNotReturn]
         protected object ThrowLiteralConversionError(GraphQLParser.AST.GraphQLValue input, string? description = null) { }
+        [System.Diagnostics.CodeAnalysis.DoesNotReturn]
         protected object ThrowSerializationError(object? value) { }
+        [System.Diagnostics.CodeAnalysis.DoesNotReturn]
         protected object ThrowValueConversionError(object? value) { }
-        public virtual GraphQLParser.AST.GraphQLValue? ToAST(object? value) { }
+        public virtual GraphQLParser.AST.GraphQLValue ToAST(object? value) { }
```

Comment:
Either returns object or throws + each usage looks like `blabla.ToAST() ?? throw new ...`

Reply:
I mean that each method actually never returns `null` (though it _can_).